modules/EmailMan/EmailMan.php

```diff
@@ -631,19 +644,31 @@ public function sendEmail($mail, $save_emails = 1, $testmode = false)
             $GLOBALS['log']->fatal('Encountered invalid email address: ' . $module->email1 . " Emailman id=$this->id");
             return true;
         }
-        
-        if (
-                $module instanceof Basic &&
-                $sugar_config['email_enable_confirm_opt_in'] &&
-                !$module->fromSugarEmailAddressField('email1')->confirm_opt_in
-        ) {
-            global $log;
-            $log->warn('An email is not confirmed opt in: '. $module->email1);
+
+        if ($this->shouldBlockEmail($module)) {
+            $GLOBALS['log']->debug('Email Address was sent due to not being confirm opt in' . $module->email1);
```

Comment:
Why log level changed to `warn` to `debug` here?

Reply:
update the to error level